pymongo/network.py

```diff
@@ -161,9 +163,26 @@ def command(
 
     if max_bson_size is not None and size > max_bson_size + message._COMMAND_OVERHEAD:
         message._raise_document_too_large(name, size, max_bson_size + message._COMMAND_OVERHEAD)
-
+    command_logger = logging.getLogger("pymongo.command")
+    # TODO: add serverConnectionId
```

Comment:
Is this todo resolved? Or can you link the PYTHON ticket?

Reply:
Yes, resolved: https://jira.mongodb.org/browse/PYTHON-2822